packages/terra-date-input/tests/wdio/date-input-spec.js

```diff
@@ -360,7 +360,7 @@ Terra.describeViewports('Date Input', ['medium'], () => {
         Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
 
         browser.click('input[name="terra-date-year-date-input"]');
-        browser.keys(['9999', 'ArrowUp']);
+        browser.keys(['2100', 'ArrowUp']);
```

Comment:
We should also add a test to test entering a year outside of 1900 and 2100.

Reply:
Added test